sfdx-project.schema.json

```diff
@@ -553,6 +560,21 @@
       "title": "Post Install Script",
       "description": "The post install script."
     },
+    "packageDirectory.seedMetadata": {
+      "type": "object",
+      "title": "Seed Metadata",
+      "description": "Metadata not meant to be packaged, but deployed before deploying packaged metadata",
```

Comment:
does this text appear on CLI help? if yes, then it must be reviewed with Doc writers

Reply:
No, the schema isn't used when displaying CLI help text.  That would come from (in this case) plugin-packaging.